static/ui.js

```diff
@@ -77,10 +77,23 @@ document.querySelector('#check-website-url').onclick = function(e) {
   xhr.onload = function() {
     if (this.response.error) {
       _log(this.response.error, 'error');
+      if (this.response.manifestUrl) {
+        _log('Manifest URL: ' + this.response.manifestUrl);
+      }
     } else {
-      ManifestParser.parse(this.response.content);
-      _showParserLogs({manifestUrl : this.response.manifestUrl});
-      _showParserTips();
+      var manifestUrl = new URL(this.response.manifestUrl);
+      var websiteUrl = new URL(this.response.websiteUrl);
+      if (manifestUrl.origin == websiteUrl.origin ||
+          (this.response.originHeader &&
+          (this.response.originHeader.includes(websiteUrl.origin) ||
```

Comment:
nit: one space here to make it clear, this is part of the same condition as the line above?